### ISSUE TITLE
PSPNet implementation

### ISSUE THREAD

**Issue by thawro** [qubvel/segmentation_models.pytorch#805]
I found a few inconsistencies with the [original paper](https://arxiv.org/pdf/1612.01105.pdf)
1. There is no dilation strategy for encoders for `PSPNet` in `smp`, while authors of the paper used dilated strategy for the encoder (i.e. ResNet):
> "...we use a pretrained ResNet [13] model with the **dilated network strategy** [3, 40] to extract the feature map."
2. In the `smp` there is a dropout layer before segmentation head, while authors of the paper do not mention dropout at all (maybe because batch normalization is used)
3. The convolutional layer before segmentation head uses `kernel_size=1` and the convolutional layer inside segmentation head uses `kernel_size=3`. Shouldnt that be `kernel_size=1` for segmentation head and `kernel_size=3` for conv before head?

Are those modifications intentional? If so, what was the reason?
